certman-operator-services/certman-operator.yaml

```diff
@@ -0,0 +1,15 @@
+services:
+- hash: 0a5e2b891727d5996421a3ca5efe91f746c75d7f 
+  name: certman-operator
+  path: /hack/olm-registry/olm-artifacts-template.yaml
+  url: https://github.com/openshift/certman-operator
```

Comment:
let's wait until this exists before merging

Reply:
Everything is merged and ready on the certman side